### DISCUSSION TITLE
Bug services d'annuaire

### DISCUSSION THREAD

**Discussion by Francisco-Ordas** [dbarzin/mercator#494] [Q&A]
Bonjour
Après avoir saisi un premier objet de type "Services d'annuaire d'administration", j'obtiens systématiquement une erreur 500 sur l'accès à l'URL : <Domaine>/admin/annuaires
Est-ce un bug connu ?
Correction ou contournement possible en intervenant dans la BDD ?
Merci pour le retour
Francisco

**Comment by dbarzin** [comment]
Ce bug a été corrigé il y a quelques jours. Il faut mettre à jour l'application.
